src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -42,139 +44,153 @@ class HaAutomationPicker extends LitElement {
+          title: this.hass.localize(
+            "ui.panel.config.automation.picker.headers.name"
+          ),
+          sortable: true,
+          filterable: true,
+          direction: "asc",
+          grows: true,
+          template: (name, automation: any) => html`
+            ${name}
+            <div class="secondary">
+              ${this.hass.localize("ui.card.automation.last_triggered")}:
+              ${automation.attributes.last_triggered
+                ? formatDateTime(
+                    new Date(automation.attributes.last_triggered),
+                    this.hass.language
+                  )
+                : this.hass.localize("ui.components.relative_time.never")}
+            </div>
+          `,
+        },
+      };
+      if (!narrow) {
+        columns.excecute = {
+          title: "",
+          template: (_info, automation) => html`
+            <mwc-button .automation=${automation} @click=${this._excecute}>
+              ${this.hass.localize("ui.card.automation.trigger")}
+            </mwc-button>
+          `,
+        };
+      }
+      columns.info = {
+        title: "",
+        type: "icon-button",
+        template: (_info, automation) => html`
+          <paper-icon-button
+            .automation=${automation}
+            @click=${this._showInfo}
+            icon="hass:information-outline"
+            title="${this.hass.localize(
+              "ui.panel.config.automation.picker.show_info_automation"
+            )}"
+          ></paper-icon-button>
+        `,
+      };
+      columns.edit = {
+        title: "",
+        type: "icon-button",
+        template: (_info, automation: any) => html`
+          <paper-icon-button
+            .icon=${automation.attributes.id
+              ? "hass:pencil"
+              : "hass:pencil-off"}
+            .disabled=${!automation.attributes.id}
+            title="${this.hass.localize(
+              "ui.panel.config.automation.picker.show_info_automation"
+            )}"
+          ></paper-icon-button>
+          ${!automation.attributes.id
+            ? html`
+                <paper-tooltip position="left">
+                  ${this.hass.localize(
+                    "ui.panel.config.automation.picker.only_editable"
+                  )}
+                </paper-tooltip>
+              `
+            : ""}
+        `,
+      };
+      return columns;
+    }
+  );
+
   protected render(): TemplateResult {
     return html`
-      <hass-tabs-subpage
+      <hass-tabs-subpage-data-table
         .hass=${this.hass}
         .narrow=${this.narrow}
         back-path="/config"
         .route=${this.route}
         .tabs=${configSections.automation}
+        .columns=${this._columns(this.narrow, this.hass.language)}
+        .data=${this._automations(this.automations)}
+        id="entity_id"
+        .noDataText=${this.hass.localize(
+          "ui.panel.config.automation.picker.no_automations"
+        )}
+        @row-click=${this._editAutomation}
```

Comment:
I don't think that this is a good experience if there are multiple actions on the row to be able to click on the row too. Means you click next to the execute, you go to edit mode.

Reply:
I think a double click on the row would be a good experience for getting into the more info dialog along with the (i) icon